src/mixins/hideLabelOnDrag.js

```diff
@@ -7,6 +7,9 @@ export default {
       }
 
       this.shape.attr('label/display', 'none');
+      this.shape.getEmbeddedCells().forEach(element => {
+        element.attr('label/display', 'none');
```

Comment:
Could limit this to boundary events

Reply:
Guess I'd have to do it with a filter on:
```
element.attributes.type === 'processmaker.components.nodes.boundaryEvent.Shape'
```

For now I just hid label if it had text.